src/base/bittorrent/session.cpp

```diff
@@ -1808,14 +1841,14 @@ void Session::processShareLimits()
             }
 
             if (torrent->seedingTimeLimit() != TorrentHandle::NO_SEEDING_TIME_LIMIT) {
-                const int seedingTimeInMinutes = torrent->seedingTime() / 60;
-                int seedingTimeLimit = torrent->seedingTimeLimit();
+                const auto seedingTimeInMinutes = std::chrono::duration_cast<std::chrono::minutes>(torrent->seedingTime());
+                std::chrono::minutes seedingTimeLimit = torrent->seedingTimeLimit();
                 if (seedingTimeLimit == TorrentHandle::USE_GLOBAL_SEEDING_TIME)
                      // If Global Seeding Time Limit is really set...
                     seedingTimeLimit = globalMaxSeedingMinutes();
 
-                if (seedingTimeLimit >= 0) {
-                    qDebug("Seeding Time: %d (limit: %d)", seedingTimeInMinutes, seedingTimeLimit);
+                if (seedingTimeLimit >= seedingTimeLimit.zero()) {
```

Comment:
if you just want a duration of 0 minutes, you can also say `0min` (as long as you `using namespace std::chrono_literals`).

Reply:
We still support C++11, there are no chrono literals there.

Reply:
sort of. user defined literals has to start with an underscore, so you could have `_s` and `_min` for seconds and minutes

Reply:
And that is why I would better switch to C++14 instead of creating slightly different copies of its content in qBt.